src/utils/Chart.js

```diff
@@ -0,0 +1,42 @@
+const d3 = require('d3');
+
+const Chart = {
+  getDataMinMaxValues (data, axis) {
+    let max = d3.max(data, d => {
+      return d[axis];
+    });
+
+    let min = d3.min(data, d => {
+      return d[axis];
+    });
+
+    const maxDigits = Math.ceil(max).toString().length - 1;
+    const multiplier = Math.pow(10, maxDigits);
+
+    max = Math.ceil(max / multiplier) * multiplier;
+    min = Math.floor(min / multiplier) * multiplier;
```

Comment:
Do you need a custom multiplier for the min too? Let's say the min value was 959 and max value was 36254. Then you would have 40000 and 10000. Wouldn't you want 959 be 1000?